packages/components/psammead-figure/README.md

```diff
@@ -0,0 +1,52 @@
+# psammead-figure &middot; [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/BBC-News/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/psammead-figure.svg)](https://www.npmjs.com/package/@bbc/psammead-figure) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/BBC-News/psammead/blob/latest/CONTRIBUTING.md)
+
+## Description
+
+The `psammead-figure` component is a styled `<figure>` element.
+
+## When to use this component
+
+Use this component when a `<figure>` element, which represents self-contained content that frequently has a caption, is required. Figures usually wrap an image.
+
+The following example shows `psammead-figure` wrapping an
+* [psammead-image-placeholder](https://github.com/BBC-News/psammead/tree/latest/packages/components/psammead-image-placeholder), a `<div>` with background image
+* [psammead-image](https://github.com/BBC-News/psammead/tree/latest/packages/components/psammead-image), an `<img>`
+* [psammead-caption](https://github.com/BBC-News/psammead/tree/latest/packages/components/psammead-caption), a `<figcaption>`
+* [psammead-visually-hidden-text](https://github.com/BBC-News/psammead/tree/latest/packages/components/psammead-visually-hidden-text), a hidden `<span>` available only to assitive technology
+
+```jsx
+import Caption from '@bbc/psammead-caption';
+import Figure from '@bbc/psammead-figure';
+import Image from '@bbc/psammead-image';
+import ImagePlaceholder from '@bbc/psammead-image-placeholder';
+import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
+
+const Wrapper = ({ src, alt, ratio, captionText }) => (
+  <Figure>
+    <ImagePlaceholder ratio={ratio}>
+      <Image alt={alt} src={src} />
+    </ImagePlaceholder>
+    <Caption>
+      <VisuallyHiddenText>Image caption, </VisuallyHiddenText>
+      {captionText}
+    </Caption>
+  </Figure>
+);
+
```

Comment:
```suggestion
```
(remove newline)

Reply:
We are inconsistent about having a last line of blank space in our package READMEs. Could we tidy this up in a separate issue?

Reply:
Yes but this is a code block, not an empty trailing line for the entire README.

Code blocks shouldn't have trailing lines 👍